### PR TITLE
Remove spammy log line in case of failing to send the log

### PR DESCRIPTION
**Pull request by sitalkedia**
### Description

We see this log line spamming the validator logs in case of network error. Removing this log line.

### CODE REVIEW

**Review by zekun000 — APPROVED** [review]

**Comment by zekun000** [comment]
you probably want to squash the lint commits before merge

**Comment by sitalkedia** [comment]
> you probably want to squash the lint commits before merge

I thought the auto-merge thing already does the squashing? @geekflyer Can you confirm if that's the case?

**Comment by geekflyer** [comment]
> > you probably want to squash the lint commits before merge
> 
> I thought the auto-merge thing already does the squashing? @geekflyer Can you confirm if that's the case?

next to the auto-merge button there is a selector that allows you to choose the merge strategy: either rebase or squash. In this case you should use squash. https://www.notion.so/aptoslabs/GitHub-Flow-4decc76691db40da8d475a88b8f1f276#cb776cf5f6a54757ac8e547c92e9ae45 also calls that out more explicitly.
Personally I use squash 90% of the time - the commit message links also better to the PR when you use squash.

**Comment by sherry-x** [comment]
Can you cherry-pick it into testnet branch?

**Pull request merged**
